grid2op/Opponent/BaseOpponent.py

```diff
@@ -73,3 +73,23 @@ def attack(self, observation, agent_action, env_action, budget, previous_fails):
         # TODO information, and forward something to the "act" method.
         attack = self._do_nothing
         return attack
+
+    def tell_attack_continues(self, observation, agent_action, env_action, budget):
+        """
+        The purpose of this method is to tell the agent that his attack is being continued.
```

Comment:
and to indicate the current state of the grid.

At every time step, either "attack" or "tell_acttack_continues" is called exactly once.

Reply:
(^détail de la doc)

Reply:
done